parlai/agents/rag/retrievers.py

```diff
@@ -17,10 +17,7 @@
 import torch.nn
 from tqdm import tqdm
 
-try:
-    from transformers import BertTokenizerFast as BertTokenizer
-except ImportError:
-    from transformers import BertTokenizer
+from transformers import BertTokenizer
```

Comment:
Is there a better way to make this change? Ideally we'd continue using `BertTokenizerFast` when it's available, since it is **significantly** faster

Reply:
Indeed. Given the try-catch, what is wrong with this?

Reply:
For some reason in fbcode ImportError was not thrown in this step but rather in the step below. Brought this back and added try catch in the section below to handle that.